internal/handlers/handlers.go

```diff
@@ -1 +1,75 @@
 package handlers
+
+import (
+	"fmt"
+	"io"
+
+	
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+	"encoding/json"
+	"github.com/Serikkambarov/6sprint-f/internal/service"
+)
+
+func IndexHandler(w http.ResponseWriter, r *http.Request) {
+	http.ServeFile(w, r, "index.html")
+}
+
+func UploadHandler(w http.ResponseWriter, r *http.Request) {
+	err := r.ParseMultipartForm(10 << 20) // 10MB
+	if err != nil {
+		http.Error(w, "Ошибка парсинга формы", http.StatusInternalServerError)
```

Comment:
Это клиент что-то не так прислал
```suggestion
		http.Error(w, "Ошибка парсинга формы", http.StatusBadRequest)
```